<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GZDoom launcher by Fres</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0" colspan="3">
     <widget class="QTextEdit" name="textEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: black; color: green;</string>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::NoTextInteraction</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="killButton">
        <property name="text">
         <string>Zabij proces</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="runButton">
        <property name="text">
         <string>Zacznij grę</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupGameType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Typ</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="radioServer">
           <property name="text">
            <string>Serwer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioClient">
           <property name="text">
            <string>Klient</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxBD">
           <property name="text">
            <string>Brutal Doom</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupWADs">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>IWAD</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QListWidget" name="IWADListWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>153</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="PWADgroupBox">
        <property name="title">
         <string>PWAD</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_7"/>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="refreshButton">
        <property name="text">
         <string>Odśwież</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Adresy IP:</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="labelIPHost">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupOptionsClient">
          <property name="title">
           <string>Opcje klienta</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="labelIP">
             <property name="text">
              <string>Adres IP:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditIP">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="maxLength">
              <number>18</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupOptionsServer">
          <property name="title">
           <string>Opcje serwera</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="4" column="1">
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Poziom trudności:</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QRadioButton" name="radioEasy">
                <property name="text">
                 <string>I'm too young to die</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioMedium">
                <property name="text">
                 <string>Hey, not too rough</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioHard">
                <property name="text">
                 <string>Hurt me plenty</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioUltra">
                <property name="text">
                 <string>Ultra-Violence</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioUltraUltra">
                <property name="text">
                 <string>Nightmare</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>Poziom:</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="QLabel" name="labelEp">
                <property name="text">
                 <string>Epizod:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="labelMap">
                <property name="text">
                 <string>Mapa:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="spinBoxEp">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>99</number>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QSpinBox" name="spinBoxMap">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>20</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Tryb gry:</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QRadioButton" name="radioCoop">
                <property name="text">
                 <string>Kooperacja</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioDm">
                <property name="text">
                 <string>Deathmatch</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QGroupBox" name="groupBox_5">
             <property name="title">
              <string>Ilość graczy:</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QSpinBox" name="spinBoxPlayers">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>8</number>
                </property>
                <property name="value">
                 <number>2</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="title">
         <string>Dodatkowe polecenia:</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QLineEdit" name="lineEditArgs"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>744</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionSettings"/>
    <addaction name="actionRefresh"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuInformacje">
    <property name="title">
     <string>Informacje</string>
    </property>
    <addaction name="actionAutor"/>
   </widget>
   <addaction name="menuPlik"/>
   <addaction name="menuInformacje"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Wyjdź</string>
   </property>
  </action>
  <action name="actionAutor">
   <property name="text">
    <string>Autor</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>Uruchom</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Odśwież</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Ustawienia</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
